### PR TITLE
fixes the trailing zero bytes trimming problem

### PR DESCRIPTION
**Pull request by xinstein**
Addresses #280 

replaces #282

### CODE REVIEW

File: clickhouse_driver/columns/numpy/stringcolumn.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ class NumpyByteStringColumn(NumpyColumn):
     null_value = b''
 
     def read_items(self, n_items, buf):
-        return np.array(buf.read_strings(n_items), dtype=self.dtype)
+        # return np.array(buf.read_strings(n_items), dtype=self.dtype)
```
Comment (xzkostyan) [inline clickhouse_driver/columns/numpy/stringcolumn.py:29]: What `dtype` was before this fix in array instance?
Do we need this comment?

**Review by xzkostyan — CHANGES_REQUESTED** [review]
We should write test for this bugfix. `tests/numpy/columns/test_string.py` is a good place for it.

**Comment by xinstein** [comment]
I can't run the test case locally due to a lot of importing errors, I looked into some and it's due to my OS (windows10).

**Comment by xzkostyan** [comment]
Test is not working. Feel free to reopen PR.

**Pull request closed without merging**